plugins/prow-job/commands/analyze-test-failure.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Analyzes test errors from console logs and Prow CI job artifacts
+argument-hint: Pass prowjob-url as $1, the test name as $2.
```

Comment:
argument-hint is what you see in the cli when you tab the command. Can we align with this format https://github.com/search?q=repo%3Aopenshift-eng%2Fai-helpers%20argument-hint&type=code?